javascript/code-challenges/challenges-01.test.js

```diff
@@ -0,0 +1,227 @@
+"use strict";
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named `addOne` that takes an array of numbers, and returns a new array of the numbers, incremented by 1.
+
+Use `forEach` to loop over the input array and work with each value.  Push the new value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  // Solution code here...
+  let arr2 = [];
+  arr.forEach((n) => {
+    arr2.push(n + 1);
+  });
+  return arr2;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named `addExclamation` that takes an array of strings, and returns a new array of the same strings with an "!" added to the end.
+
+Use `forEach` to loop over the input array. Modify each string, and add the updated value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addExclamation = (arr) => {
+  // Solution code here...
+  let arr2 = [];
+  arr.forEach((n) => {
+    arr2.push(!);
```

Comment:
Here you are trying to push just an exclamation mark into your arr2, We just want to add that ! to the end of each string